app/models/micropost.rb

```diff
@@ -6,6 +6,7 @@ class Micropost < ApplicationRecord
   validate :picture_size
 
   scope :descending, ->{order(created_at: :desc)}
+  scope :of_following, ->(following_ids, id){where "user_id IN (?) or user_id = ?", following_ids, id}
```

Comment:
scope này nhóm params lại 1 mảng ids và where user_id: ids thôi e
in rồi or gì thành 2 cái cho hiệu suất tệ hơn